packages/react/src/components/MapCard/MapCard.mdx

```diff
@@ -464,4 +464,4 @@ const MapboxDragPanelExample = ({ data }) => {
 ## Feedback
 
 Help us improve this component by providing feedback, asking questions on Slack, or updating this file on
-[GitHub](https://github.com/carbon-design-system/carbon-addons-iot-react/tree/next/packages/react/src/components/MenuButton/README.md).
+[GitHub](https://github.com/carbon-design-system/carbon-addons-iot-react/tree/next/packages/react/src/components/MenuButton/MenuButton.mdx).
```

Comment:
This is actually my old copy/paste error :-) It should link to the MapCard not the MenuButton

Reply:
fixed.